src/Components/ImagesTable/ImagesTable.tsx

```diff
@@ -173,54 +169,13 @@ const ImagesTable = () => {
   return (
     <>
       <>
-        <DeleteBlueprintModal
-          setShowDeleteModal={setShowDeleteModal}
-          isOpen={showDeleteModal}
+        <ImagesTableToolbar
+          itemCount={itemCount}
```

Comment:
Let's use our Redux `useAppSelector` hook to get all of this data in the component instead of passing props.

Reply:
Ugh? I don't know what you mean, I could probably calculate the itemCount here if I get the composes, but that's about it, everything else is local state in parent component :(

What we **could** do is to have pagination as children and do

```jsx
<ImagesTableToolbar>
  <Pagination ...>
</ImagesTableToolbar>
```

Reply:
Oh, I got confused, this is the pagination related to the images table. I was thinking about Amir's PR that added these properties to the _blueprints_ yesterday. My bad. 

Maybe we should do something similar and add an `imagesSlice` so that the way we handle pagination is consistent.

Anyway, my bad, disregard. :sweat_smile: